### PR TITLE
[icons] feat: "intel" icon

### PR DESCRIPTION
**Pull request by gredman2018**
#### Checklist

- [x] Includes tests
- [x] Update documentation

#### Changes proposed in this pull request:

Added a the new intelligence icon with 16px and 20px

#### Reviewers should focus on:

Everything renders correctly at both sizes!

#### Screenshot

Screenshot 2023-01-19 at 10 39 38 AM

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @gredman2018! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/icons/icons.json [hunk 1]
```diff
@@ -3960,5 +3960,14 @@
         "tags": "circle, cross mark, fail, delete, no, close, remove, not, available, unavailable, wrong, incorrect, cancel",
         "group": "action",
         "codepoint": 62262
+    },
+
```
Comment (adidahiya) [inline packages/icons/icons.json]: extra newline here is not allowed:

```suggestion
```

File: packages/icons/icons.json [hunk 2]
```diff
@@ -3960,5 +3960,14 @@
...
+    {
+        "displayName": "Intelligence",
+        "iconName": "intelligence",
+        "tags": "intelligence, intel, observe, watch, capture, humint",
+        "group": "interface",
```
Comment (adidahiya) [inline packages/icons/icons.json]: is this really an interface icon? let's throw it in misc for now

```suggestion
        "group": "miscellaneous",
```

**Review by adidahiya — CHANGES_REQUESTED** [review]
@CPerinet to confirm design approval 👍🏽 / 👎🏽 ?

File: packages/icons/icons.json [hunk 3]
```diff
@@ -3960,5 +3960,13 @@
...
+    },
+    {
+        "displayName": "Intelligence",
+        "iconName": "intelligence",
+        "tags": "intelligence, intel, observe, watch, capture, humint",
+        "group": "miscellaneous",
+        "codepoint": 62263
     }
 ]
+
```
Comment (adidahiya) [inline packages/icons/icons.json]: oops, missed this one

```suggestion
```

**Review by adidahiya — CHANGES_REQUESTED** [review]

**Review by adidahiya — APPROVED** [review]

**Comment by adidahiya** [comment]
sorry about the failing deploy-preview job -- I'll fix that -- here's the docs preview link: https://output.circle-artifacts.com/output/job/7ebf1edd-d362-4194-9fda-f8170c8baec9/artifacts/0/packages/docs-app/dist/index.html#icons

image

**Comment by gredman2018** [comment]
Hi! When/how can I fix circleci deploy-preview failing?

**Comment by adidahiya** [comment]
deploy-preview is returning a failing exit code because it's building on a fork and doesn't have the necessary environment variables. That's not a problem with your PR, just an issue with the preview script right now. It should exit gracefully with a successful exit code -- I'm going to fix that in a separate PR. In the meantime, I'll merge this PR, it's good to go :shipit:

**Pull request merged**